app/docs/0.14.x/plugin-development/entities-cache.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Plugin Development - Caching custom entities
+book: plugin_dev
+chapter: 7
+---
+
+# {{page.title}}
+
+#### Modules
+
+```
+kong.plugins.<plugin_name>.daos
```

Comment:
This used to be `daos` and `hooks` from the last plugin dev guide before the new cache mechanism). `daos` is kept here since `hooks` are removed, and most likely plugin authors want to cache their custom entities (defined in `daos`). A custom entity (in a custom DAO) is used in the example in this page.